src/accelerate/logging.py

```diff
@@ -85,6 +85,9 @@ def get_logger(name: str, log_level: str = None):
 
     ```python
     >>> from accelerate.logging import get_logger
+    >>> from accelerate import Accelerator
+
+    >>> accelerator = Accelerator()
```

Comment:
This is incorrect. We only need to instantiate before we log. Just making the logger should not require this, and the current example docstring is accurate. If it’s wrong in some way, we need to fix that.

Reply:
You are right, it is possible to call `get_logger` without initializing the `Accelerator` first, but not to use it, like calling `logger.info("My log")`. Therefore, the example as is does not work. Of course, we could move the new lines below the call to `get_logger`, but I wanted to keep the imports grouped for better readability.

Reply:
Import grouped together is fine :)

Reply:
So you mean the import can stay there but `accelerator = Accelerator()` should be moved below `logger = get_logger(__name__)`?

Reply:
Done